### ISSUE TITLE
ComboButton as a secondary button

### ISSUE THREAD

**Issue by pmweeks98** [carbon-design-system/ibm-cloud-cognitive#2049]
## What package(s) are you using?

- [X] Carbon for IBM Products (or Carbon for IBM Cloud & Cognitive)
- [ ] Legacy/CDAI
- [ ] Legacy/Security

## What did you want to know?
Combobutton is always rendered as a primary button
![image](https://user-images.githubusercontent.com/56260662/174293966-79f9bf4e-4b78-4cfd-bc1a-6636689dbec2.png)

Is there any way of rendering it as secondary as below without using some css trickery?
![image](https://user-images.githubusercontent.com/56260662/174293913-c9d1154e-d4e4-4c4e-ba52-28aaa38f6eaa.png)

**Comment by matthewgallo** [comment]
Hey @pmweeks98, we will need to double check with the design for this component. @Laura-Marshall do you have thoughts on allowing for this behavior?

**Comment by matthewgallo** [comment]
As I'm reviewing the guidance for the component, it looks like we've intentionally not provided this option.

> Since the Carbon guidelines state “Secondary buttons may only be used in conjunction with a primary action” the menu button is not available as a secondary button.

This guidance can be found [here](https://pages.github.ibm.com/syd/pal-1/components/button-menu/).

**Comment by Laura-Marshall** [comment]
Yeah that's correct. As the combo button use is dictated by one of the options being defined as primary, having the button in any other form would go against its usage. For a less prominent menu, a tertiary or ghost menu button is available :)
